lib/entities/climate.ts

```diff
@@ -6,95 +6,103 @@
  * @license Apache License 2.0, see LICENSE for more details.
  */
 
-import { CommandHandler, TYPES as ENTITYTYPES } from "./entity.js";
-import Entity from "./entity.js";
+import { CommandHandler, Entity, EntityType, EntityName } from "./entity.js";
 import log from "../loggers.js";
 
 // Climate entity states
-enum STATES {
-  UNAVAILABLE = "UNAVAILABLE",
-  UNKNOWN = "UNKNOWN",
-  OFF = "OFF",
-  HEAT = "HEAT",
-  COOL = "COOL",
-  HEAT_COOL = "HEAT_COOL",
-  FAN = "FAN",
-  AUTO = "AUTO"
+export enum States {
+  Unavailable = "UNAVAILABLE",
+  Unknown = "UNKNOWN",
+  Off = "OFF",
+  Heat = "HEAT",
+  Cool = "COOL",
+  HeatCool = "HEAT_COOL",
+  Fan = "FAN",
+  Auto = "AUTO"
 }
 
 // Climate entity features
-enum FEATURES {
-  ON_OFF = "on_off",
-  HEAT = "heat",
-  COOL = "cool",
-  CURRENT_TEMPERATURE = "current_temperature",
-  TARGET_TEMPERATURE = "target_temperature",
-  TARGET_TEMPERATURE_RANGE = "target_temperature_range",
-  FAN = "fan"
+export enum Features {
+  OnOff = "on_off",
+  Heat = "heat",
+  Cool = "cool",
+  CurrentTemperature = "current_temperature",
+  TargetTemperature = "target_temperature",
+  TargetTemperatureRange = "target_temperature_range",
+  Fan = "fan"
 }
 
 // Climate entity attributes
-enum ATTRIBUTES {
-  STATE = "state",
-  CURRENT_TEMPERATURE = "current_temperature",
-  TARGET_TEMPERATURE = "target_temperature",
-  TARGET_TEMPERATURE_HIGH = "target_temperature_high",
-  TARGET_TEMPERATURE_LOW = "target_temperature_low",
-  FAN_MODE = "fan_mode"
+export enum Attributes {
+  State = "state",
+  CurrentTemperature = "current_temperature",
+  TargetTemperature = "target_temperature",
+  TargetTemperatureHigh = "target_temperature_high",
+  TargetTemperatureLow = "target_temperature_low",
+  FanMode = "fan_mode"
 }
 
 // Climate entity commands
-enum COMMANDS {
-  ON = "on",
-  OFF = "off",
-  HVAC_MODE = "hvac_mode",
-  TARGET_TEMPERATURE = "target_temperature",
-  TARGET_TEMPERATURE_RANGE = "target_temperature_range",
-  FAN_MODE = "fan_mode"
+export enum Commands {
+  On = "on",
+  Off = "off",
+  HvacMode = "hvac_mode",
+  TargetTemperature = "target_temperature",
+  TargetTemperatureRange = "target_temperature_range",
+  FanMode = "fan_mode"
 }
 
 // Climate entity device classes
-enum DEVICECLASSES {}
+export enum DeviceClasses {}
 
 // Climate entity options
-enum OPTIONS {
-  TEMPERATURE_UNIT = "temperature_unit",
-  TARGET_TEMPERATURE_STEP = "target_temperature_step",
-  MAX_TEMPERATURE = "max_temperature",
-  MIN_TEMPERATURE = "min_temperature",
-  FAN_MODES = "fan_modes"
+export enum Options {
+  TemperatureUnit = "temperature_unit",
+  TargetTemperatureStep = "target_temperature_step",
+  MaxTemperature = "max_temperature",
+  MinTemperature = "min_temperature",
+  FanModes = "fan_modes"
+}
+
+export enum TemperatureUnit {
+  Celsius = "C",
+  Fahrenheit = "F"
```

Comment:
These temperature unit values are not correct. See:
- https://github.com/unfoldedcircle/core-api/blob/main/doc/entities/entity_climate.md#options
- https://github.com/unfoldedcircle/core-api/blob/main/integration-api/UCR-integration-asyncapi.yaml#L1814-L1819

Note: the old unit test was using the incorrect "C" value. This worked because the option values were not restricted.

Reply:
I changed it.